client/cache/error_code.go

```diff
@@ -123,6 +123,16 @@ var InvalidSsdbCacheValue = berror.DefineCode(4002022, moduleName, "InvalidSsdbC
 SSDB cache only accept string value. Please check your input.
 `)
 
+var InvalidInitParameters = berror.DefineCode(4002024, moduleName, "InvalidInitParameters", `
+Invalid init cache parameters.
```

Comment:
The read through pattern already use the code 4002024, so you need to use a new one. 😄